extensions/shared/external-media/media-browser/media-item.js

```diff
@@ -18,16 +18,23 @@ function MediaItem( props ) {
 		}
 	}, [ props.onClick ] );
 
-	// Catch space and enter key presses.
-	const onKeyDown = event => {
-		if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
-			// Prevent spacebar from scrolling the page down.
-			event.preventDefault();
-			onClick( event );
-		}
-	};
+	const onKeyDown = useCallback(
+		event => {
+			// Catch space and enter key presses.
+			if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
```

Comment:
This whole handler could also be moved to parent component. No need to handle each item individually.

Reply:
See https://github.com/Automattic/jetpack/pull/16118#discussion_r438716986

Reply:
Is it me or will this function only be recomputed if `props.onKeyDown` changes? Would that mean that it will always memorze the first instance of its invocation? I think perhaps it only "works" because the `onKeyDown` is probably a different reference on each render. However if this was itself memorized then this could start to cause issues.

Reply:
Thinking about this perhaps it just returns the same function _reference_. However it doesn't memoize the _result_ of calling the function so maybe we're ok.